meetings/2022-11-21.md

```diff
@@ -0,0 +1,111 @@
+# W3C Solid Community Group: Data Interoperability Panel
+
+* Date: 2022-11-21T15:00:00Z
+* Call: https://meet.jit.si/solid-data-interoperability
+* Chat: https://gitter.im/solid/data-interoperability-panel
+* Repository: https://github.com/solid/data-interoperability-panel
+
+## Present
+
+* elf Pavlik
+* Wouter Termont
+* Laurens Debackere
+* Justin Bingham
+
+### Regrets
+
+* ...
+
+---
+
+## Announcements
+
+### New meetings schedule
+
+eP: bi-weekly on Mondays at 15:00UTC => next meeting on 2022-11-21T15:00:00Z
```

Comment:
"Next meeting" was scheduled for the date of *this* meeting, being minuted here...

Best practice would be to include the dates of both Previous Meeting and Next Meeting in the minutes for each meeting, optimally linking Previous to Minutes, and Next to Agenda.